sdl-core/src/test/scala/io/smartdatalake/testutils/TestUtil.scala

```diff
@@ -120,7 +120,7 @@ object TestUtil extends SmartDataLakeLogger {
                      )(implicit instanceRegistry: InstanceRegistry, context: ActionPipelineContext): HiveTableDataObject = {
     val table = Table(db=db,name=tableName,primaryKey=primaryKeyColumns)
     val path = dirPath+s"$tableName"
-    val hTabDo = HiveTableDataObject(id=s"${tableName}DO",path=Some(path),schemaMin=schemaMin.map(SparkSchema),table=table)
+    val hTabDo = HiveTableDataObject(id=s"${tableName}DO",path=Some(path),schemaMin=schemaMin.map(SparkSchema.apply),table=table)
```

Comment:
in case your wondering why I am adding SparkSchema.apply: it makes it much clearer that actually a function, the "constructor" is passed as an argument and not the class SparkSchema itself. During Refactoring I had created a companion object SparkSchema and encountered an error as it tried to pass the companion object here. Took me a while to understand that you actually want to pass the constructor function. So let's just do that, less confusing and if we ever create a companion object it will save somebody some time